py-torch: allow v0.4.1 to build despite missing nervanagpu repo (#14370)
spack/spack: lib/spack/docs/packaging_guide.rst

* ``tag``: Name of a tag to fetch.
* ``commit``: SHA hash (or prefix) of a commit to fetch.
* ``submodules``: Also fetch submodules recursively when checking out this repository.

> * ``submodules_delete``: An list of submodules to forcibly delete from the repository

Reviewer: ```suggestion
* ``submodules_delete``: A list of submodules to forcibly delete from the repository
```

Author: Done

> * ``submodules_delete``: An array of submodules to forcibly delete from the repository

Reviewer: ```suggestion
* ``submodules_delete``: A list of submodules to forcibly delete from the repository
```
In reality, it's any iterable, but list is the most common iterable in Python.
